decorator.rb

```diff
@@ -0,0 +1,12 @@
+require './nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
```

Comment:
- [ ] Why are you running super here? the parent class doesn't have a constructor